services/blockchain-connector/shared/sdk/dynamicReward.js

```diff
@@ -36,6 +40,22 @@ const getRewardTokenID = async () => {
 	return rewardTokenID;
 };
 
+const getDefaultRewardAtHeight = async height => {
+	try {
+		// TODO: Update endpoint once exposed by sdk
+		const defaultReward = await invokeEndpoint('dynamicReward_getDefaultRewardAtHeight', { height });
```

Comment:
This is also missing the error handling to switch to `reward_*****` endpoint.

Reply:
https://github.com/LiskHQ/lisk-service/pull/1377#discussion_r1043418768